package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@jet-lab/jet-engine",
-  "version": "0.2.2",
+  "version": "0.2.3",
```

```suggestion
  "version": "0.2.2",
```
This should be changed by `npm version patch` which also pushes a new tag to the repository once PR has landed on `master`.